### PR TITLE
Upgrade message unit tests

### PR DESCRIPTION
**Pull request by DenisRybas**

### CODE REVIEW

File: x/dclupgrade/types/message_approve_upgrade_test.go [hunk 1]
```diff
@@ -22,22 +23,41 @@ func TestMsgApproveUpgrade_ValidateBasic(t *testing.T) {
 				Name:    testconstants.Plan.Name,
 			},
 			err: sdkerrors.ErrInvalidAddress,
```
[the remark this answers is not in the document]
Comment (DenisRybas) [reply x/dclupgrade/types/message_approve_upgrade_test.go:25]: fixed

File: x/dclupgrade/types/message_propose_upgrade_test.go [hunk 2]
```diff
@@ -22,22 +52,64 @@ func TestMsgProposeUpgrade_ValidateBasic(t *testing.T) {
 				Plan:    testconstants.Plan,
 			},
 			err: sdkerrors.ErrInvalidAddress,
```
[the remark this answers is not in the document]
Comment (DenisRybas) [reply x/dclupgrade/types/message_propose_upgrade_test.go:69]: fixed

File: x/dclupgrade/types/message_propose_upgrade_test.go [hunk 3]
```diff
@@ -22,22 +52,64 @@ func TestMsgProposeUpgrade_ValidateBasic(t *testing.T) {
...
-		}, {
-			name: "valid address",
+		},
+		{
+			name: "Plan len 0",
```
[the remark this answers is not in the document]
Comment (DenisRybas) [reply x/dclupgrade/types/message_propose_upgrade_test.go]: fixed

File: x/dclupgrade/types/message_propose_upgrade_test.go [hunk 4]
```diff
@@ -22,22 +52,64 @@ func TestMsgProposeUpgrade_ValidateBasic(t *testing.T) {
...
+			err: sdkerrors.ErrInvalidRequest,
+		},
+		{
+			name: "Plan height less than 0",
+			msg: MsgProposeUpgrade{
+				Creator: sample.AccAddress(),
+				Plan:    planHeightLess0,
+			},
+			err: sdkerrors.ErrInvalidRequest,
+		},
```
[the remark this answers is not in the document]
Comment (DenisRybas) [reply x/dclupgrade/types/message_propose_upgrade_test.go:102]: added

**Review by spivachuk — APPROVED** [review]

**Pull request merged**